src/P2P/MulticastBroadcaster.ts

```diff
@@ -0,0 +1,193 @@
+
+import PeerStore from '@polykey/PeerStore/PeerStore'
+import KeyManager from '@polykey/KeyManager'
+import RPCMessage from '@polykey/RPC/RPCMessage'
+import dgram from 'dgram'
+import crypto from 'crypto'
+import { EventEmitter } from 'events'
+
+// This module is based heavily on libp2p's mDNS module:
+// https://github.com/libp2p/js-libp2p-mdns
+// It is supposed to discover peers on the local network
+// This module was also generated with the help of:
+// https://nrempel.com/using-udp-multicast-with-node-js/
+//
+// """
+// In computer networking, the multicast DNS (mDNS) protocol
+// resolves hostnames to IP addresses within small networks
+// that do not include a local name server
+// """
+
+const UDP_MULTICAST_PORT = 5353;
```

Comment:
Who made this port choice? We can do it as an environment variable? Although I guess we can hard code our own "service" registry.

Reply:
Sure we can make it an environment variable, I think that would work well for both electron and nativescript

Reply:
These have been made primarily environment variables with defaults set to what they were before.